### DISCUSSION TITLE
Redmi note 5 pro unsupported???

### DISCUSSION THREAD

**Discussion by Weegley** [AppGround-io/bluetooth-keyboard-and-mouse-support#223] [Bugs & Problems]
Why Redmi note 5 pro got unsupported? What changed an why?
When you buy premium version you expect that it will work.

**Comment by AppGround-io** [comment]
Unfortunately not all devices have the required software support for the app to work, but you can use the free version of the app to find out if the device is supported.

**Reply by Weegley** [reply]
May be you don't understand the question.
1. I paid for software.
2. It was working good.
3. Device still the same, but not supported any-more and can't install/update from Play Store.
4. Download and install apk directly.
5. Working again. From what we can assume that this device still have required support?

So the question is: What really changed? Why paid software becomes unusable in official way, when in real it still works?

BTW proposition to use free version to users who already paid is....

**Reply by AppGround-io** [reply]
Thank you for the clarification.
For the Redmi Note 5 Pro there has been a lot of reports in the past that the device doesn't work, that's why I disabled it a few month ago.
Since it seems to work for you I reactivated it and you should be able to download it again.

**Reply by Weegley** [reply]
All features present in paid version work well. Can make proof video :)
Anyway - thx, and sorry, if i was too rude.
